test/tautotrace.m

```diff
@@ -217,35 +294,88 @@ function testMultipleInstances(testCase)
+            testCase.applyFixture(matlab.unittest.fixtures.PathFixture(examplefolder));
+            
             % create and instance and then clear
-            at = opentelemetry.autoinstrument.AutoTrace(@example1); %#ok<NASGU>
+            at = opentelemetry.autoinstrument.AutoTrace(@linearfit_example); %#ok<NASGU>
             clear("at")
 
             % create a new instance should not result in any error
-            at = opentelemetry.autoinstrument.AutoTrace(@example1); %#ok<NASGU>
+            at = opentelemetry.autoinstrument.AutoTrace(@linearfit_example); %#ok<NASGU>
         end
 
         function testInvalidInputFunction(testCase)
             % testInvalidInputFunction: negative test for invalid input
 
+            % add the example folders to the path
+            examplefolder = fullfile(fileparts(mfilename('fullpath')), "autotrace_examples", "linearfit_example");
+            testCase.applyFixture(matlab.unittest.fixtures.PathFixture(examplefolder));
+            
             % anonymous function
-            verifyError(testCase, @()opentelemetry.autoinstrument.AutoTrace(@()example1), "opentelemetry:autoinstrument:AutoTrace:AnonymousFunction");
+            verifyError(testCase, @()opentelemetry.autoinstrument.AutoTrace(@()linearfit_example), "opentelemetry:autoinstrument:AutoTrace:AnonymousFunction");
 
             % builtin function
             verifyError(testCase, @()opentelemetry.autoinstrument.AutoTrace(@uplus), "opentelemetry:autoinstrument:AutoTrace:BuiltinFunction");
 
             % nonexistent function
             verifyError(testCase, @()opentelemetry.autoinstrument.AutoTrace(@bogus), "opentelemetry:autoinstrument:AutoTrace:InvalidMFile");
         end
+
+        function testAutoManualInstrument(testCase)
+            % testAutoManualInstrument: using both auto and manual
+            % instrumentation
+            
+            % add the example folders to the path
+            examplefolder = fullfile(fileparts(mfilename('fullpath')), "autotrace_examples", "manual_instrumented_example");
+            testCase.applyFixture(matlab.unittest.fixtures.PathFixture(examplefolder));
+
+            % set up AutoTrace
+            at = opentelemetry.autoinstrument.AutoTrace(@manual_instrumented_example);
+
+            % run the example
+            [~] = beginTrace(at, 100);
+
+            % perform test comparisons
+            results = readJsonResults(testCase);
+            verifyNumElements(testCase, results, 6);
+
+            % check tracer and span names
+            tracername = "ManualInstrument";
+            verifyEqual(testCase, string(results{1}.resourceSpans.scopeSpans.scope.name), tracername); 
+            verifyEqual(testCase, string(results{1}.resourceSpans.scopeSpans.scope.name), tracername); 
+            verifyEqual(testCase, string(results{1}.resourceSpans.scopeSpans.scope.name), tracername); 

```

Comment:
Is this check meant to be repeated like this?

Reply:
Good catch. I neglected to update the indices after copying and pasting. Corrected now.